incubator/jaeger/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: 1.12.0
+appVersion: 1.13.0
```

Comment:
Is removing the configmaps a breaking change for users using the current version?
If so version should be 2.0.0 and maybe the chart should be moved to stable at the same time.

Reply:
It is not a breaking change to my knowledge. In any case this is the app version. The chart version is still <1.0.0. The chart will make its way to stable soon enough but not quite in this PR.